doc/sphinx/about.rst

```diff
@@ -0,0 +1,97 @@
+.. _sec-about:
+
+*************
+About Cantera
+*************
+
+Cantera is an open source scientific computing project, that allows users to
+quickly and efficiently incorporate and execute fundamental thermodynamic,
+chemical kinetic, and species transport calculations in their work. This can be
+achieved by standalone command-line calls to core Cantera functions,
```

Comment:
I'm not sure I understand what is meant by "command-line calls", at least as far as what Cantera currently has.

Reply:
How about `achieved by interactive calls to core...`?